amt/wsman.py

```diff
@@ -36,12 +38,13 @@
     'cd': 'Intel(r) AMT: Force CD/DVD Boot',
 }
 
-
+FRIENDLY_POWER_STATE = {v: k for (k, v) in six.iteritems(POWER_STATES)}
 
 def friendly_power_state(state):
-    for k, v in POWER_STATES.items():
-        if v == int(state):
-            return k
+    try:
```

Comment:
I put the `try .. except` here to be compatible with the previous behaviour, although IMHO, it is better to raise an exception rather than return `None`. It will also simplify the code?

What say you?